src/App.jsx

```diff
@@ -1,33 +1,64 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
-import './App.css'
+import { useState } from "react"
+import "./styles.css"
+import NewToDoForm from "./NewToDoForm"
+import ToDoList from "./ToDoList"
+
 
 function App() {
-  const [count, setCount] = useState(0)
+  // Here I am using the state to add items to a list (which is in this case an array)
+  const [toDos, setToDos] = useState([])
+
+  function addToDo(title) {
+    // Now I add a to-do to the list(array)...
+        setToDos((currentToDos) => {
+          return [
+            ...currentToDos,  
+            // ...with a unique identifier (see key in the <li>) that needs to be created BEFORE it is rendered
+            { id: crypto.randomUUID(),
+            title,
+            completed: false },
+        ]
```

Comment:
mind the indentation, knowing where those brackets start-end makes things easy to understand.

If you are coding using VSCode installing [prettier](https://marketplace.visualstudio.com/items?itemName=esbenp.prettier-vscode) will help you with this and other code smells :)